tests/api/presenters/test_login_member_presenter.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional
+from uuid import UUID, uuid4
+
+from arbeitszeit.use_cases.log_in_member import LogInMemberUseCase
+from arbeitszeit_web.api_presenters.interfaces import JsonBoolean, JsonDict
+from arbeitszeit_web.api_presenters.login_member_api_presenter import (
+    LoginMemberApiPresenter,
+)
+from arbeitszeit_web.api_presenters.response_errors import Unauthorized
+from arbeitszeit_web.session import UserRole
+from tests.api.presenters.base_test_case import BaseTestCase
+from tests.session import FakeSession
+
+
+class TestViewModelCreation(BaseTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.presenter = self.injector.get(LoginMemberApiPresenter)
+        self.session = self.injector.get(FakeSession)
+
+    def test_correct_error_raises_if_wrong_mail_adress_was_given(self) -> None:
```

Comment:
I would prefer if you specified the error type already in the test name. `correct_error` is kind of vague.